README.md

```diff
@@ -101,8 +120,14 @@ What are the differences between calling and referencing a function? Please prov
 Using the object literal notation, Define an object called student and give it the properties (your attributes) of name, age, and a method sayHello, that outputs "Hi, my name is [your_name]".
 
 Your Answer:
-```js
-// write code here
+```
+var student = {
+  name: 'cam',
```

Comment:
Yes, this is correct! in your sayHello method, I might include `"Hi, my name is " +  this.name` over hardcoding in your name